app/views/members/show.html.erb

```diff
@@ -62,6 +63,21 @@
               <% end %>
             </p>
           </div>
+					
+          <div class="px-inline-only-2">
+            <h4 class="heading strong-500">
+							L'adhésion prend fin le	
+						</h4>
+
+						<hr class="mt-0 mb-0">
+
+            <p class="text-lg-7 line-height-1_8 mt-4">
+							<% if @member.end_subscription.present? %>
+								<%= @member.end_subscription.strftime('%d-%m-%Y') %>
+							<% end %>
+						</p>
+					</div>
+
```

Comment:
This information should be private, and must not be shown in `members/show.rb`.
Only admins, in the admin interface, should be able to see this information about other members.
You can delete this section